wiki/Glossary/en.md

```diff
@@ -319,7 +319,7 @@ These panels are sortable by various attributes.
 Users that are using the osu!client are displayed, not ones that are participating in chat from using a IRC client.
 The user panels supply lots of information on each user.
 If the mouse cursor is placed on them.
-The colour of the user panels indicates the user's current activity; an explaination for these codes, see [Chat Console](/wiki/Chat-Console/en.md#panels).
+The colour of the user panels indicates the user's current activity; an explaination for these codes, see [Chat Console](/wiki/Chat-Console/#panels).
```

Comment:
should be Chat_Console, and the #panels link doesn't work anymore so you can remove